broker_rabbit/channels.py

```diff
@@ -88,11 +92,22 @@ def consume_one_message(self):
             self.on_message(self._channel, method, header, body)
             return True
 
+    def bind_callback(self, raw_message):
+        try:
+            message = json.loads(raw_message)
```

Comment:
This can raise an error. Catch the error and put the message on the disk with readable message.